### ISSUE TITLE
Could you provide selected data for each dataset?

### ISSUE THREAD

**Issue by swj0419** [yaoxingcheng/TLM#2]
Great work! I was wondering if the selected data for each dataset will be available soon. Thanks

**Comment by yaoxingcheng** [comment]
Thank you for your interesting in our work. All task datasets and corresponding selected data have been uploaded to Huggingface. Take [AGNews](https://huggingface.co/datasets/yxchar/ag-tlm/tree/main) for example (Links to the rest of the datasets are shown in `README.md`), you can find two splits called `small_external.csv` and `large_external.csv` in the uploaded files. Those are the selected data both on Wiki+Book and on Wiki+Book+CCNews+OpenWeb+Stories.

**Comment by swj0419** [comment]
Thanks! Do you retrieve data for examples only in the task training set or the combination of task training, dev, and test set?

**Comment by yaoxingcheng** [comment]
We retrieve data only for the training set.
